web/frontend/src/pages/election/components/utils/useChangeAction.tsx

```diff
@@ -366,8 +384,12 @@ const useChangeAction = (
         }
       };
 
+      // TODO: can be modified such that if the majority of the node are
+      // initialized than the election status can still be set to initialized
       const promises = Array.from(nodeProxyAddresses.values()).map((proxy) => {
-        return initialize(proxy);
+        if (proxy !== '') {
+          return initialize(proxy);
+        }
```

Comment:
It seems that the linter would be happier with an explicit `return undefined` if proxy is empty :)